### DISCUSSION TITLE
LXQt wiki and search machines

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#2301] [General]
What I always suspected I learned today:
https://github-wiki-see.page/

> [GitHub prevents crawling of repository wiki content](https://github.com/github/feedback/discussions/4992) through [HTTP headers](https://github.com/github/feedback/discussions/4992#discussioncomment-1448177). The blocking greatly reduces the effectiveness of content of pages on affected Wikis if they cannot be searched on search engines such as Google or DuckDuckGo.

[Test](https://www.qwant.com/?q=LXQt+does+not+provide+its+own+window+manager+but+any+arbitrary+window+manager) with an excerpt.

The question is if we add a link in that way:

> If you would like to greatly increase the chances and ranking of your GitHub Wiki content being indexed, consider adding a link to your content on GHWSEE to a location where search engines can index. For an example of this in action, see how the [MiSTer historical computing and gaming recreation project linked a "crawlable wiki" that GHWSEE serves in their README on GitHub which can be indexed](https://github.com/MiSTer-devel/Main_MiSTer#main_mister-main-binary-and-wiki-repo). While the link is "nofollow" on the README as displayed on GitHub, that attribute is more of a guideline than a hard rule nowadays for search engines anyway. Popular repositories can confer their own ranking power onto their proxied and indexable Wiki content on GHWSEE.

**Comment by tsujan** [comment]
I've had no problem in finding them with google or SearX+google, and I tried DuckDuckGo for a week and got disappointed in general. But that may not count.
